### PR TITLE
Document overwrite cron flag

### PR DESCRIPTION
**Pull request by adelelopez**
This includes documentation for https://github.com/pachyderm/pachyderm/pull/3509 for the upcoming 1.8.6 release.

### CODE REVIEW

File: doc/reference/pipeline_spec.md [hunk 1]
```diff
@@ -507,6 +509,12 @@ times from the past or, skip times from the present and only start running
 on matching times in the future. Times should be formatted according to [RFC
 3339](https://www.ietf.org/rfc/rfc3339.txt).
 
+`input.cron.overwrite` is a flag to specify whether you want the timestamp file
+to be overwritten on each tick. It is optional, and if not specified it will
+default to simply writting new files each tick. What this means is that by
```
Comment (jdoliner) [inline doc/reference/pipeline_spec.md]:
```suggestion
default to simply writing new files each tick. What this means is that by
```

File: doc/reference/pipeline_spec.md [hunk 2]
```diff
@@ -507,6 +509,12 @@ times from the past or, skip times from the present and only start running
...
+default, pachd will process only the new data since the previous tick on each
+tick. If true, pachd will reprocess _all_ the data on each tick.
```
Comment (jdoliner) [inline doc/reference/pipeline_spec.md]: I'd reword this a little bit to say:
```
by default, pachd expects only the new information to be written out for each tick, and will combine that data with the data from previous ticks. If `"overwrite"` is set to `true` it expects the full dataset to be written out for each tick and will replace previous outputs with the news data written out.

You wording suggests to me that pachd is doing more than it actually is.

Comment (pappasilenus) [reply doc/reference/pipeline_spec.md]: Agree with @jdoliner 's comment.
